Lib/ufo2ft/outlineCompiler.py

```diff
@@ -1460,6 +1464,8 @@ def setupOtherTables(self):
         self.setupTable_glyf()
         if self.ufo.info.openTypeGaspRangeRecords:
             self.setupTable_gasp()
+        if self.compileTrueTypeHinting:
+            self.setupTable_cvt()
```

Comment:
it's confusing to read an option called compileTrueTypeHinting whose effect is to only compiling the CVT table and not the rest of the hinting tables. I'd remove the option altogether. If there are TRUETYPE_INSTRUCTIONS_KEY in the ufo you build whatever you are requested to build, otherwise you don't.

Reply:
I agree and changed the behaviour in #577